### ISSUE TITLE
Bug: Element not closed when clicked outside or tabbed out

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-ngx#923]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.

When clicked outside of the element or tabbed out, the component should be closed. 
- Popover
- Dropdown
- Multi input
- Search input

**Comment by MattL75** [comment]
The clicking part is already true. For the tabbing, I was under the impression that a focus trap should be implemented in the body of these components, with the escape key to get out of it.
